pages/events/index.vue

```diff
@@ -183,12 +178,12 @@ export default class extends QiskitPage {
   }
 
   // multiselect
-  regionOptions = this.getOptions(this.regions)
-  typeOptions = this.getOptions(this.types)
-  theme: string = 'light'
-  labelRegions: string = 'Locations'
-  labelTypes: string = 'Types'
-  feedback: string = 'fixed'
+  regionsOptions = this.getOptions(this.regions)
+  typesOptions = this.getOptions(this.types)
+  regionsLabel: string = 'Locations'
+  typesLabel: string = 'Types'
+  regionsFilters: string = 'regionFilters'
+  typesFilters: string = 'typeFilters'
```

Comment:
You have created two variables for these texts, so you can use them on lines 44-47 and 62-64. This way, if we changes the text, we only have to replace it in one part of the code